front/src/utils/consts.js

```diff
@@ -94,6 +94,10 @@ export const GetWeatherModes = {
   DailyForecast: 'dailyForecast'
 };
 
+export const GetTodolistStatus = {
```

Comment:
Can't we use the `RequestStatus` enum instead of adding a new one?

Reply:
Yes, I can